proto/minder/v1/minder.proto

```diff
@@ -1004,8 +996,7 @@ message Provider {
 // Context defines the context in which a rule is evaluated.
 // this normally refers to a combination of the provider, organization and project.
 message Context {
-    string provider = 1;
-    optional string organization = 2;
+    optional string provider = 1;
```

Comment:
According to https://protobuf.dev/programming-guides/proto3/#field-labels:

> If no explicit field label is applied, the default field label, called “implicit field presence,” is assumed. (You cannot explicitly set a field to this state.) A well-formed message can have zero or one of this field (but not more than one). You also cannot determine whether a field of this type was parsed from the wire. An implicit presence field will be serialized to the wire unless it is the default value. For more on this subject, see [Field Presence](https://protobuf.dev/programming-guides/field_presence).

This means that if a client built with `optional string provider` sends an empty message to a server built with `string provider`, the server will read the string as `""`.  Similarly, if the server sends `""` as a `string provider`, the client will read `null`, because the default value of the field won't be serialized.

Reply:
Thanks for confirming this.